webviz_subsurface/_datainput/eclipse_init_io/pvt_gas.py

```diff
@@ -286,6 +335,52 @@ def viscosity_unit(self, latex: bool = False) -> str:
             return fr"${unit_system.viscosity().symbol}$"
         return f"{unit_system.viscosity().symbol}"
 
+    def density_unit(self, latex: bool = False) -> str:
+        """Creates and returns a string containing the unit symbol of the density.
+
+        Args:
+            latex: True if the unit symbol shall be returned as LaTeX, False if not.
+
+        Returns:
+            A string containing the unit symbol of the density.
+
+        """
+        unit_system = EclUnits.create_unit_system(
+            self.original_unit_system
+            if self.keep_unit_system
+            else EclUnitEnum.ECL_SI_UNITS
+        )
+
+        if latex:
+            return fr"${unit_system.density().symbol}$"
+        return f"{unit_system.density().symbol}"
+
+    def ratio_unit(self, latex: bool = False) -> str:
+        """Creates and returns a string containing the unit symbol of the phase ratio.
+
+        Args:
+            latex: True if the unit symbol shall be returned as LaTeX, False if not.
+
+        Returns:
+            A string containing the unit symbol of the phase ratio.
+
+        """
+        unit_system = EclUnits.create_unit_system(
+            self.original_unit_system
+            if self.keep_unit_system
+            else EclUnitEnum.ECL_SI_UNITS
+        )
+
+        if latex:
+            return (
+                fr"${{{unit_system.surface_volume_liquid().symbol}}}/"
+                fr"{{{unit_system.surface_volume_gas().symbol}}}$"
```

Comment:
Is this one of the occasions where _not_ using `f`-strings are more readable? Not sure what is best/most readable wrt. seeing the extra LaTeX symbols.
```suggestion
                "${" + unit_system.surface_volume_liquid().symbol + "}/{" + unit_system.surface_volume_gas().symbol + "}$"
```

Reply:
Not a big fan of the `+` connections but I see your point. Most people are probably not used to escaping curly brackets.